install.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We could probably give this file a more descriptive name, but I can do that after merging.

Reply:
OK, I'll leave this one with you :)